FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1453,4 +1453,81 @@ public function testCategoryNamesWithSlashesAreExportedCorrectly($articleConfigu
             $cat_urls
         );
     }
+
+    public function productsWithPseudoSalesProvider()
+    {
+        return [
+            'Pseudo sale value of 0 is not exported' => [
+                'pseudoSales' => 0,
+                'expectedSalesFrequency' => 1
```

Comment:
Where is `1` being set? I do not see any sales being created in the test.

Reply:
In the test article that is created there are by default an order with one item with `quantity: 1` .